websocket.go

```diff
@@ -220,18 +223,18 @@ func (c *Client) Send(dst *network.ServerIdentity, path string, buf []byte) ([]b
 		if err != nil {
 			return nil, NewClientError(err)
 		}
-		c.si = dst
+		c.connections[dest] = conn
 	}
 	defer func() {
 		if !c.keep {
-			c.Close()
+			c.Close(dest)
```

Comment:
You can:
1. return a slice of errors (which might be empty)
2. return a concatenated string of errors
3. return the first error and then either stop on error or go on anyway
4. only `log.Error` the errors
5. Ignore the errors altogether

Who cares if a connection-*close* has an error - the *opening* error is important, as is the *writing* error. But not the closing error. So I propose to:
- `log.Error` an eventual error in the *closing* part in `Client.Send` - this will log the error on stderr, but make the `Client.Send` method succeed *if* the open and write succeeded, of course.
- return a concatenated string of errors (or nil if no error) in `Client.Close`